.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/ChineseDurationParserConfiguration.cs

```diff
@@ -82,6 +97,78 @@ public List<DateTimeParseResult> FilterResults(string query, List<DateTimeParseR
             return candidateResults;
         }
 
+        private DateTimeResolutionResult ParseMergedDuration(string text, DateObject referenceTime)
+        {
+            var ret = new DateTimeResolutionResult();
+            var durationExtractor = DurationExtractor;
+
+            // DurationExtractor without parameter will not extract merged duration
+            var ers = durationExtractor.Extract(text, referenceTime);

```

Comment:
This seems to be duplicating extraction work, which suggest it would be better to have the merge happen at the parser level.